src/pages/uses/automotive/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Enact for Automotive
+github: https://github.com/enactjs/docs/blob/develop/src/pages/uses/automotive/index.md
+---
+
+Web technologies are well suited for developing rich applications for the automotive market. Enact makes it easy for large organizations to implement performant, reliable and robust applictions for automotive. Its features include:
+
+* Comprehensive component set, begin creating apps immediately
```

Comment:
The wording of these bullets is a bit awkward with the first phrase completing the preamble above but the second phrase sort of standing on its own. I'll admit that I'm oversensitive to parallelism for lists so maybe not a big deal.

Reply:
No, I usually am, too.  So, I'll clean this up.